demos/appyx-interactions/android/src/main/kotlin/com/bumble/appyx/interactions/sample/SpotlightExperiment.kt

```diff
@@ -43,7 +43,6 @@ fun SpotlightExperiment(
     modifier: Modifier = Modifier,
     orientation: Orientation = Orientation.Horizontal,
     reverseOrientation: Boolean = false,
-    visualisation: (UiContext) -> BaseVisualisation<Target, SpotlightModel.State<Target>, *, *>
```

Comment:
I wouldn't remove this, but have +1 parameter for state.

Reply:
At first I tried that, but that will imply creating the SpotlightModel outside as well, which means also having to extract the `items` list - I can do it but it seemed wrong in my personal opinion. WDYT?

Reply:
Ah right. Any other way to keep the visualisation parameterised from outside? Maybe a simple enum.